CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### Unreleased
   - Fix handling of None responses in remote call
-
+  - Changed status_code to status in cora_processor.py and request_helper.py for SDX logging
+  - Changed 'nack for retry' to 'nack' in consumer.py for SDX logging
```

Comment:
Try to keep in imperative, present tense: e.g. "Change" rather than "Changes" or "Changed"

Where something is "code", ideally use the backtick markdown: e.g. "Change to `nack`" rather than "Change to 'nack'"

Keep a blank newline between the end of the current block and the next version/date tag.